### PR TITLE
[s]Bubblegum and Colossus

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
I split this off from the lavaland russians because I can't motivate myself to finish large things without seeing any of the progress in game.

Still missing dead sprites and loot, and the code is a complete mess, but I  opened for balance testing/test merging

### CODE REVIEW

File: code/modules/projectiles/projectile.dm [hunk 1]
```diff
@@ -49,6 +49,7 @@
 	var/stamina = 0
 	var/jitter = 0
 	var/forcedodge = 0 //to pass through everything
+	var/projectile_speed = 1 //1 is the minimum, and the fastest. The higher the number, the slower the projectile
```
Comment (RemieRichards) [inline code/modules/projectiles/projectile.dm]: look a few lines above kor, there's already a speed var.

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+	layer = 6.1
+	luminosity = 2
+	var/target
+
+/obj/effect/at_shield/New()
+	..()
+	spawn()
+		for(var/i in 1 to 80-1)
+			if(target)
+				loc = get_turf(target)
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: need to change this to orbiting

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+/mob/living/simple_animal/hostile/megafauna/colossus/Destroy()
+	qdel(internal)
+	. = ..()
+
+/obj/effect/at_shield
+	name = "anti-toolbox field"
+	desc = "A shimmering forcefield protecting the colossus."
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "at_shield2"
+	layer = 6.1
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: use define here

**Comment by TechnoAlchemisto** [comment]
him big

**Comment by GunHog** [comment]
Pics???

**Comment by optimumtact** [comment]
well at least try to finish one of them kor

**Comment by KorPhaeron** [comment]
I added a commit 2 hours ago you bully

**Pull request closed without merging**